packages/node/src/index.ts

```diff
@@ -17,7 +17,22 @@ export default class LogtoClient extends BaseClient {
   constructor(config: LogtoConfig, adapter: Pick<ClientAdapter, 'navigate' | 'storage'>) {
     super(config, {
       ...adapter,
-      requester: createRequester(fetch),
+      requester: createRequester(
+        config.appSecret
+          ? async (...args: Parameters<typeof fetch>) => {
+              return fetch(args[0], {
+                ...args[1],
+                headers: {
+                  ...args[1]?.headers,
+                  authorization: `basic ${Buffer.from(
+                    `${config.appId}:${config.appSecret ?? ''}`,
```

Comment:
Will config.appSecret be false as the entry condition is a truthy config.appSecret?

Reply:
Would it be better to apply client secret authorization only when `appSecret` is not empty?
```
...conditional(config.appSecret && ({ authorization: `basic ${ ... }` }))
```

Reply:
in line 21:
```
config.appSecret
?
```